test/integration/imagechange_buildtrigger_test.go

```diff
@@ -477,12 +479,21 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 		if err != nil {
 			t.Fatalf("Couldn't create Image: %v", err)
 		}
+
+		var newBuild *buildapi.Build
+		var event watchapi.Event
 		// wait for initial build event from the creation of the imagerepo
-		event := <-watch.ResultChan()
+		for {
+			event = <-watch.ResultChan()
+			newBuild = event.Object.(*buildapi.Build)
+			if _, exists := ignoreBuilds[newBuild.Name]; !exists {
```

Comment:
it's really ignore because it's a list of builds for which we want to ignore the events.

if it were seenBuilds i'd expect to add a build to that list as soon as we saw it, not when we're done with it (and want to start ignoring it).

but i will add a comment explaining the intent/behavior.